hw2/05-form.js

```diff
@@ -0,0 +1,25 @@
+// Add your code here
+
+function formSubmit(event) {
+  let fullName = event.target[0].value;
+  let email = event.target[1].value;
+  let registration = event.target[2].value;
+  let courseOne = event.target[3].value;
+  let courseTwo = event.target[4].value;
+  let courseThree = event.target[5].value;
+  let extra = event.target[6].value;
```

Comment:
You could omit all of these allocations and use the values directly in the template literals below. Alternatively, you could use `id` attributes to refer to the form's values by name:
```
const form = document.getElementById('form-name');
...
console.log(`Name: ${form['full-name'].value}\n` + ...);
```